app/src/components/fineTunes/ContentTabs/TestingData/TestingDataRow.tsx

```diff
@@ -0,0 +1,140 @@
+import { Th, Td, Thead, Tr, Text, VStack } from "@chakra-ui/react";
+import { type ChatCompletionMessage } from "openai/resources/chat";
+import SyntaxHighlighter from "react-syntax-highlighter";
+
+import { type RouterOutputs } from "~/utils/api";
+
+export const TableHeader = () => {
+  return (
+    <Thead>
+      <Tr>
+        <Th w="30%">Input</Th>
+        <Th w="35%">Original Output</Th>
+        <Th>Fine-Tuned Model Output</Th>
+      </Tr>
+    </Thead>
+  );
+};
+
+type TestingEntry = RouterOutputs["datasetEntries"]["listTestingEntries"]["entries"][number];
+
+const TestingDataRow = ({
+  prunedInput,
+  output,
+  datasetEntry: { output: originalOutput },
+  errorMessage,
+}: {
+  prunedInput: TestingEntry["prunedInput"];
+  output: TestingEntry["output"];
+  outputTokens: number | null;
+  datasetEntry: TestingEntry["datasetEntry"];
+  errorMessage?: string;
+}) => {
+  return (
+    <Tr
+      alignItems="flex-start"
+      sx={{
+        td: { verticalAlign: "top" },
+      }}
+    >
+      <Td>
+        <FormattedInput prunedInput={prunedInput} />
+      </Td>
+      <Td
+        sx={{
```

Comment:
Just curious, does this not work with inline Chakra properties?

Reply:
Inline Chakra properties work fine here. This syntax is an artifact of the process I went through during initial styling, in which I was adding properties to the Tr to be shared across at Td elements. I'll put the styling back in inline Chakra properties for simplicity now.